sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -292,6 +300,26 @@ public static boolean isConfigured() {
         return getInstance().isInstanceConfigured();
     }
 
+    /**
+     * Check whether app is running in App Center Test.
+     *
+     * @return true if running in App Center Test, false otherwise
+     * (and where no test dependencies in release.
+     */
+    public static boolean isRunningInAppCenterTestCloud() {
+        try
+        {
```

Comment:
Please follow formatting to style

Reply:
@owenniblock, please follow SDK coding/formatting guideline at https://msmobilecenter.visualstudio.com/Mobile-Center/_wiki/wikis/Mobile-Center.wiki?wikiVersion=GBwikiMaster&pagePath=%2FHome%2FTeams%2FSDKs%2FGet%20started%20with%20SDK%20development%2FAndroid%20SDK%20extra%20topics&pageId=2135&anchor=before-pushing-your-changes when using Android Studio. Thank you.